Automattic/jetpack
# [JS Package]Connection: add manage connection dialog component

Completes the first task of #26577

#### Changes proposed in this Pull Request:

- Create the manage connection dialog component's skeleton and story
<img width="1773" alt="Screen Shot 2022-10-06 at 16 55 33" src="https://user-images.githubusercontent.com/16329583/194347224-1fffb22e-4961-441d-b988-e7fb4321efa8.png">

Note: The component will not be used yet. it will be styled in a follow-up PR.
You'll find the detailed steps [here](https://github.com/Automattic/jetpack/issues/26577).

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Build and open the storybook
* Look for the component in `JS PACKAGES / Connection / Manage Connection Dialog`
* Confirm that the component is there, and that both controls are available (isOpen, title)
